script/commands/SetConfig.s.sol

```diff
@@ -1,147 +0,0 @@
-/* solhint-disable gas-custom-errors, check-send-result */
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.24;
-
-import {L2YnOFTAdapterUpgradeable} from "../../src/L2YnOFTAdapterUpgradeable.sol";
-import {BaseData} from "../BaseData.s.sol";
-import {BaseScript} from "../BaseScript.s.sol";
-import {UlnConfig} from "@layerzerolabs/lz-evm-messagelib-v2/contracts/uln/UlnBase.sol";
-import {OptionsBuilder} from "@layerzerolabs/lz-evm-oapp-v2/contracts/oapp/libs/OptionsBuilder.sol";
-import {ILayerZeroEndpointV2} from
-    "@layerzerolabs/lz-evm-protocol-v2/contracts/interfaces/ILayerZeroEndpointV2.sol";
-
-import {SetConfigParam} from "@layerzerolabs/lz-evm-protocol-v2/contracts/interfaces/IMessageLibManager.sol";
-
-import {console} from "forge-std/console.sol";
-
-/**
- * @notice This script bridges ynETHx tokens between chains using LayerZero OFT protocol
- *
- * @dev How it works:
- * 1. User provides destination chain ID via prompt
- * 2. If on base chain (L1):
- *    - Wraps ETH to WETH by sending ETH to WETH contract
- * 3. Bridges tokens via OFT adapter's sendFrom()
- *
- * Usage:
- * ```
- * forge script script/commands/SetConfig.s.sol:SetConfig --rpc-url <RPC_URL> --broadcast
- * ```
- */
-contract ConfigureDVNs is BaseData, BaseScript {
```

Comment:
why is this deleted?

Reply:
its not used anywhere, instead we can run the `ConfigureOFT` script that will run all the configurations for us